src/napari_experimental/group_layer_qt.py

```diff
@@ -1,16 +1,25 @@
 from __future__ import annotations
 
+import logging
+from pathlib import Path
 from typing import TYPE_CHECKING, Any
 
 from napari._qt.containers import QtNodeTreeModel, QtNodeTreeView
-from qtpy.QtCore import QModelIndex, Qt
+from napari._qt.qt_resources import get_current_stylesheet
+from qtpy.QtCore import QModelIndex, QSize, Qt
+from qtpy.QtGui import QDropEvent, QImage
 
 from napari_experimental.group_layer import GroupLayer
+from napari_experimental.group_layer_delegate import GroupLayerDelegate
 
 if TYPE_CHECKING:
     from qtpy.QtWidgets import QWidget
 
 
+logger = logging.getLogger(__name__)
+ThumbnailRole = Qt.UserRole + 2
```

Comment:
I don't think we've been using the logger up to this point - is it necessary for us to start using it here? (If so we should probably introduce it across our plugin in one go, rather than just here).

Also not sure what the `ThumbnailRole` is doing?

Reply:
Good point! The `logger` was left over from some code I copied, so I've removed this now. I've also removed the `ThumbnailRole` and I'm instead importing this value directly from napari